tests/test_codex_feed_hooks.py

```
@@ -78,6 +92,107 @@ def _run(self) -> None:
+                f"hooks codex prompt-submit failed exit={result.returncode}\n"
+                f"stdout={result.stdout}\nstderr={result.stderr}"
+            )
+
+        pids = wait_for_monitor_pids(session_id, present=True, timeout=5)
+        stop = {
+            "session_id": session_id,
+            "turn_id": turn_id,
+            "cwd": str(root),
+            "transcript_path": str(transcript_path),
+        }
+        result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)